### PR TITLE
feat(react-components): group information

### PR DESCRIPTION
**Pull request by kmaid**

### CODE REVIEW

File: packages/react-components/src/organisms/__tests__/GroupInformation.test.tsx [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+it('does not show ellipses or show more button on short descriptions', () => {
+  const { queryByText } = render(
+    <GroupInformation {...props} description={'The best group'} />,
+  );
+  expect(queryByText(/…/)).toBeNull();
+  expect(queryByText('more')).toBeNull();
+});
+
+it('shows the ellipses and show more button on long descriptions', () => {
```
Comment (jeysal) [inline packages/react-components/src/organisms/__tests__/GroupInformation.test.tsx:27]: nice 1

File: packages/react-components/src/organisms/__tests__/GroupInformation.test.tsx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  );
+  expect(getByText('The best group')).toBeVisible();
+});
+
+it('does not show ellipses or show more button on short descriptions', () => {
+  const { queryByText } = render(
+    <GroupInformation {...props} description={'The best group'} />,
+  );
+  expect(queryByText(/…/)).toBeNull();
+  expect(queryByText('more')).toBeNull();
```
Comment (jeysal) [inline packages/react-components/src/organisms/__tests__/GroupInformation.test.tsx:24]:
```suggestion
  expect(queryByText(/more/i)).toBeNull();
```

**Review by jeysal — APPROVED** [review]
There is one kinda ugly edge case that comes to my mind, due to space collapsing, which is:
```
TRIM_CONTENT=2
description='x  '
```
I believe this would show `x …\nShow more`, then `x  \nShow less`. But I wouldn't consider this important enough to handle unless anyone complains.
